src/main/java/net/sf/jabref/exporter/layout/LayoutHelper.java

```diff
@@ -80,7 +80,7 @@ public static void setCurrentGroup(String newGroup) {
         LayoutHelper.currentGroup = newGroup;
     }
 
-    private String getBracketedField(final int field) throws IOException {
+    private void getBracketedField(final int field) throws IOException {
```

Comment:
I would rename these methods since they are not returning anything and so "get" is misleading.